src/databricks/labs/remorph/helpers/reconcile_utils.py

```diff
@@ -86,7 +86,7 @@ def run(self):
 
         logger.info(f"Triggering the Job with job_id: `{reconcile_config.job_id}` ...")
 
-        wait = self._ws.jobs.run_now(job_id=reconcile_config.job_id)
+        wait = self._ws.jobs.run_now_and_wait(job_id=reconcile_config.job_id)
```

Comment:
Why do you want to wait for the run to complete?

Reply:
Will address in the Invocation PR.
 In this PR I'm targeting only Feature Implementation

Reply:
Reverted the change